lua/vgit/settings/git.lua

```diff
@@ -0,0 +1,7 @@
+local Config = require('vgit.core.Config')
+
+return Config:new({
+  cmd = 'git',
+  fallback_cwd = '',
```

Comment:
Hmm, I think we can remove the word `fallback` here and just leave it as `cwd` and `args`.

Reply:
I thought about that too but I found it confusing, people might then think these `args` are always used when in fact they are only used when the current cwd/buffer isn't a git repo, we only go to the fallback when `rev-parse --is-inside-git-dir` fails.

Reply:
That makes sense 👍🏾